translate/src/modules/user/components/UserAvatar.tsx

```diff
@@ -19,9 +21,19 @@ export function UserAvatar(props: Props): React.ReactElement<'div'> {
         rel='noopener noreferrer'
         onClick={(e: React.MouseEvent) => e.stopPropagation()}
       >
-        <Localized id='user-UserAvatar--alt-text' attrs={{ alt: true }}>
-          <img src={imageUrl} alt='User Profile' height='44' width='44' />
-        </Localized>
+        <div className='avatar-container'>
+          <Localized id='user-UserAvatar--alt-text' attrs={{ alt: true }}>
+            <img src={imageUrl} alt='User Profile' height='44' width='44' />
+          </Localized>
+          {userStatus && (
```

Comment:
Now that `userStatus` is always set, the check is no longer sufficient:

```suggestion
          {role && (
```